### PR TITLE
EVG-7249 better logging for docker host initialization

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: cloud/docker_client.go [hunk 1]
```diff
@@ -469,7 +476,6 @@ func (c *dockerClientImpl) ListContainers(ctx context.Context, h *host.Host) ([]
 	containers, err := dockerClient.ContainerList(ctx, opts)
 	if err != nil {
 		err = errors.Wrap(err, "Docker list API call failed")
-		grip.Error(err)
 		return nil, err
```
Comment (tychoish) [inline cloud/docker_client.go]: combine the error wrapping into the return.

File: cloud/docker_client.go [hunk 2]
```diff
@@ -524,7 +530,6 @@ func (c *dockerClientImpl) RemoveContainer(ctx context.Context, h *host.Host, co
 	opts := types.ContainerRemoveOptions{Force: true}
 	if err = dockerClient.ContainerRemove(ctx, containerID, opts); err != nil {
 		err = errors.Wrapf(err, "Failed to remove container '%s'", containerID)
-		grip.Error(err)
 		return err
```
Comment (tychoish) [inline cloud/docker_client.go]: combine wrap and return

File: cloud/docker_client.go (hunk 1 above)
Comment (john-m-liu) [reply cloud/docker_client.go]: fixed

File: cloud/docker_client.go (hunk 2 above)
Comment (john-m-liu) [reply cloud/docker_client.go]: fixed

**Pull request merged**